activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Loading the schema for a model that has no `table_name` raises a `NoTableError` error.
```

Comment:
```suggestion
*   Loading the schema for a model that has no `table_name` raises a `TableNotSpecified` error.
```

Reply:
Sorry this PR fell off my radar for a while; other than updating the changelog entry I think it's good to go ❤️